File: python/test/Hello.py
from pyspark import SparkContext, SparkConf

'''
Hello
'''

def main():
    # Spark Configurations
    conf = SparkConf()
    conf.set("spark.master", "local[*]")
    conf = conf.setAppName('WordCount')
    sc = SparkContext(conf=conf)
    print("Hello")

if __name__ == "__main__":
    main()

File: scala/src/main/scala/test/Hello.scala
package test

import org.apache.spark.{SparkConf, SparkContext}

/**
 * Hello
 **/
object Hello {
  def main(args: Array[String]) = {
    val sc = new SparkContext(new SparkConf().setAppName("test").setMaster("local[*]"))
    println("Hello")
    sc.stop()
  }
}
